src/nimibook/docs.nim

```diff
@@ -20,16 +41,13 @@ proc useNimibook*(nbDoc: var NbDoc) =
   # debugEcho "Current directory: ", getCurrentDir()
   # debugEcho "Output file: ", nbDoc.filename
 
-
   # templates are in nbSrcDir
   nbDoc.templateDirs = @[nbSrcDir.string]
```

Comment:
templates are staying in book folder? I thought we were moving them to a templates but maybe that was an assumption I made in my head and never shared... :)

Reply:
Maybe we misunderstood each other. My idea was to have a ``ressources/template`` folder in ``src/nimibook`` to deploy the mustache files automatically in the book folder (per the init mechanism we discussed)

Reply:
in [this comment](https://github.com/pietroppeter/nimibook/pull/19#pullrequestreview-674953517) I describe an alternative folder structure. Do you think it could be done in this PR? if not we can do it/discuss it later. If it stays like this maybe I would just change `ressources` to `resources` :)

Reply:
> on general folder structure, I was thinking, maybe we could put in src/nimibook a theme folder which contains a full example structure of a default nimibook (with folders docs\assets, templates, book, ...). If we add to that a clever config.nims (with a path ../../ we probably can also run nbook there and test the default structure during development). The theme folder is then the reference from where we populate assets.

If I understand correctly, you want to have the whole book structure in ``src/nimibook/theme`` and copy it in the current folder upon calling the ``init`` / ``update`` step ?

That is perfectly do-able, I don't think we even need a config.nims to test it (Init should do the trick).

Reply:
let's do and discuss this in another PR :)